ui.frontend/src/view/FormFieldBase.js

```diff
@@ -202,11 +202,23 @@ class FormFieldBase extends FormField {
            if (this.getDescription()) {
             ariaDescribedby += `${this.getId()}__longdescription`
           }
-           if (this.getTooltipDiv()) {
-            ariaDescribedby += ` ${this.getId()}__shortdescription`;
+          if (this.getTooltipDiv()) {
```

Comment:
Can we simplify this code and make it more readable,

`function appendDescription(descriptionType) {
    if (ariaDescribedby) {
        ariaDescribedby += ` ${this.getId()}__${descriptionType}`;
    } else {
        ariaDescribedby = `${this.getId()}__${descriptionType}`;
    }
}

if (this.getTooltipDiv()) {
    appendDescription('shortdescription');
}

if (this.getErrorDiv() && this.getErrorDiv().innerHTML) {
    appendDescription('errormessage');
}`

Reply:
Please also fix in other places, where there is code duplication